.travis.yml

```diff
@@ -41,6 +40,48 @@ matrix:
   include:
     - arch: s390x
       jdk: openjdk11
+    - dist: focal
+      group: edge
+      arch: arm64-graviton2
```

Comment:
Is it possible to use arm64 directly? 
https://docs.travis-ci.com/user/reference/overview/#linux-travisyml-examples

Reply:
Hmm,  we have tried using the common `arm64` resources, but many modules(especially Integration-Test) are unstable and easy to fail. the new added `arm64-graviton` is anounced a better performance ARM resources, so switch to use it. 
Maybe we can try back to use the `arm64` resources as we have migrated from `travis-ci.org` to `travis-ci.com` to see if the problems still occur.

Reply:
Can you link to the announcement? It looks like they all should work as indicated in the https://docs.travis-ci.com/user/reference/overview/#linux-travisyml-examples and I don't see performance concerns. 
The arm64 configuration seems more clean and simple

Reply:
There is not annoucement, just be cause I have tested many times, the ARM CI jobs are very easy to fail due to timeout errors previously. I have tested again in the PR https://github.com/apache/storm/pull/3361, it looks like now the common `arm64` resource is OK to run the ARM CI job :), so let's switch to use the `arm64` resources